### ISSUE TITLE
Perivascular extension of brain white matter lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11303]
**Preferred term label:**
Perivascular extension of brain white matter lesions

**Synonyms**
sunburst appearance

**Definition (free text, please give PubMed ID)**
https://omim.org/entry/621295

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
CADASIL1 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am wondering if this should be its own term,
Perivascular enhancement pattern
see: https://www.sciencedirect.com/science/article/pii/S015098612500001X

I cannot find this as related to CADASIL

**Comment by MickeySegal** [comment]
Oops.  It is in https://omim.org/entry/621295
"perivascular extension of white matter lesions ('sunburst appearance'), consistent with small vessel disease"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001464
- label: Cerebral perivascular enhancement pattern
- parents: [Abnormal cerebral vascular morphology (HP:0100659)](https://hpo.jax.org/app/browse/term/HP:0100659)
- pmid: PMID:39828213
- def: Focal, regional or diffuse magnetic resonance imaging (MRI) enhancement of intracranial perivascular spaces (PVS). This enhancement pattern is recognizable by its topography that follows PVS distribution, particularly identifiable in regions where PVS predominate (centra semiovalia, basal ganglia and the brachium pontis)15 although it can be found throughout the whole brain. Depending on the perpendicular or parallel orientation of MRI sections to the long axis of PVS, enhanced PVS appear as dots or curvilinear elements respectively. Depending on the causal pathology, PVE may appear more linear or nodular.
- comment: Perivascular spaces (PVS) are fluid-filled structures that form the immediate peripheral environment of small cerebral vessels. They are a central component of the glymphatic system, which plays a crucial role in maintaining cerebral homeostasis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11303
- synonyms: Perivascular enhancement pattern(NONE)
